cross/go2rtc/Makefile

```diff
@@ -27,4 +27,4 @@ include ../../mk/spksrc.cross-go.mk
 go2rtc_post_extract:
 	@$(MSG) Rename extracted folder to make imports work when used as GOPATH
 	cd $(EXTRACT_PATH) && [ -d "$(PKG_NAME)-$(PKG_VERS)" ] && mv "$(PKG_NAME)-$(PKG_VERS)" $(PKG_NAME)
-	cd "$(EXTRACT_PATH)/$(PKG_NAME)" && env $(ENV) go get github.com/tillberg/autorestart
+	cd "$(EXTRACT_PATH)/$(PKG_NAME)"
```

Comment:
This line is obsolete

```suggestion
```

### Proposal:

With the following changes, there is no need to rename the folder. 

Without POST_EXTRACT_TARGET and one level of sub folder removed, the source dir will be `$(WORK_DIR)/src/github.com/$(PKG_NAME)-$(PKG_VERS)`:


```diff
diff --git "a/cross/go2rtc/Makefile" "b/cross/go2rtc/Makefile"
index 58b25d8b..1b33a42c 100644
--- "a/cross/go2rtc/Makefile"
+++ "b/cross/go2rtc/Makefile"
@@ -4,8 +4,7 @@ PKG_EXT = tar.gz
 PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
 PKG_DIST_SITE = https://github.com/AlexxIT/go2rtc/archive
 PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
-EXTRACT_PATH = $(WORK_DIR)/src/github.com/$(PKG_NAME)
-PKG_DIR = src/github.com/$(PKG_NAME)/$(PKG_NAME)
+EXTRACT_PATH = $(WORK_DIR)/src/github.com/
 
 BUILD_DEPENDS = native/go
 
@@ -13,18 +12,10 @@ HOMEPAGE = https://github.com/AlexxIT/go2rtc/
 COMMENT  = Ultimate camera streaming application with support RTSP, RTMP, HTTP-FLV, WebRTC, MSE, HLS, MP4, MJPEG, HomeKit, FFmpeg, etc.
 LICENSE  = MIT
 
-POST_EXTRACT_TARGET = go2rtc_post_extract
-
-GO_SRC_DIR = $(EXTRACT_PATH)/$(PKG_NAME)
+GO_SRC_DIR = $(EXTRACT_PATH)/$(PKG_NAME)-$(PKG_VERS)
 GO_BIN_DIR = $(GO_SRC_DIR)/$(PKG_NAME)
 
 GO_BUILD_ARGS = -trimpath
 GO_LDFLAGS = -s -w
 
 include ../../mk/spksrc.cross-go.mk
-
-.PHONY:  go2rtc_post_extract
-go2rtc_post_extract:
-	@$(MSG) Rename extracted folder to make imports work when used as GOPATH
-	cd $(EXTRACT_PATH) && [ -d "$(PKG_NAME)-$(PKG_VERS)" ] && mv "$(PKG_NAME)-$(PKG_VERS)" $(PKG_NAME)
-	cd "$(EXTRACT_PATH)/$(PKG_NAME)"

```


just ensure the path of the license file must be adjusted in spk/go2rtc/Makefile:
```diff
diff --git "a/spk/go2rtc/Makefile" "b/spk/go2rtc/Makefile"
index 825fc8e9..3d7182d2 100644
--- "a/spk/go2rtc/Makefile"
+++ "b/spk/go2rtc/Makefile"
@@ -11,7 +11,7 @@ MAINTAINER = skrashevich
 DESCRIPTION = Ultimate camera streaming application with support RTSP, RTMP, HTTP-FLV, WebRTC, MSE, HLS, MP4, MJPEG, HomeKit, FFmpeg, etc
 DISPLAY_NAME = go2rtc
 CHANGELOG = "go2rtc updated to v1.9.4"
-LICENSE_FILE = $(WORK_DIR)/src/github.com/go2rtc/go2rtc/LICENSE
+LICENSE_FILE = $(WORK_DIR)/src/github.com/go2rtc-$(SPK_VERS)/LICENSE
 HOMEPAGE = https://github.com/AlexxIT/go2rtc
 LICENSE = MIT
 ```